style.css

```diff
@@ -0,0 +1,398 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+html {
+  height: 100%;
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  background-color: #F8F8FF;
+}
+
+h1, h3, a, p, li, label, .footer {
+  font-family: Arial;
+}
+
+.main-container {
+  margin: 0 auto;
+  max-width: 1200px;
+  -webkit-box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  -moz-box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.10);
+  width: 100%;
+  min-height:100%;
+  position:relative;
+}
+
+.head-section {
+  padding: 30px;
```

Comment:
Generally it's good practice to use `em` *OR* `px` (or `rem`) consistently for specifying values in the body. (disclaimer: it's fine to use specific pixel values to describe like `max_width` of the whole page, but then use `em` elsewhere.) 

You're going for consistency so you don't have to translate between measurement systems in your head.